packages/core/src/lavadome.mjs

```diff
@@ -41,6 +46,8 @@ export function LavaDome(host, opts) {
     const child = unselectable();
     appendChild(shadow, child);
 
+    let secret = '';
+
     function text(text) {
```

Comment:
somewhat unrelated: I find it a bit confusing with the `text(text)`, which is what, where further down. 
```suggestion
    function text(input) {
```